nowplaying/daemon.py

```diff
@@ -0,0 +1,85 @@
+import logging
+import os
+import signal
+import sys
+import time
+
+from input import observer as inputObservers
+from input.handler import InputHandler
+from track import observer as trackObservers
+from track.handler import TrackEventHandler
+
+logger = logging.getLogger("now-playing")
+
+
+class NowPlayingDaemon:
+    """The daemon part of the nowplaying app runs at all time and coordinates the input and output."""
```

Comment:
This class contains the business logic that used to be in cli.py. Unless noted the code hasn't been refactored yet.